.gitignore

```diff
@@ -8,3 +8,4 @@ phantomjsdriver.log
 .DS_Store
 test/test-results.xml
 npm-debug.log
+Chrome_*
```

Comment:
Just curious what creates `Chrome_*` files

Reply:
Our unit tests create those, at least for me.

Reply:
Top level? Not under `/test` or `/test/tmp`?  Weird.

Reply:
Yeah, top level.

```
~/git/origin-web-common master
❯ ls -l
total 616
drwxr-xr-x    3 sam  staff      96 Mar  6 08:09 Chrome_64.0.3282_(Mac_OS_X_10.13.3)
...
```